esiclient/v1/network_subnet.py

```diff
@@ -0,0 +1,137 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+import logging
+
+from osc_lib.cli import parseractions
+from osc_lib.command import command
+from osc_lib.i18n import _
+
+
+def _get_attrs_network(parsed_args):
+    attrs = {}
+    if parsed_args.network_name and \
+            parsed_args.network_name is not None:
```

Comment:
This check is redundant (if `passed_args.network_name` is `None`, the first test will fail). You could just write:

```
if parsed_args.network_name:
    attrs["name"] = parsed_args.network_name
```

You can make the same change to several other conditionals in this file.

Reply:
Fixed them!